nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexSurefireExecution.java

```diff
@@ -102,6 +118,10 @@ private Xpp3Dom addExcludedGroups(Xpp3Dom configNode) {
             if ("excludedGroups".equals(config.getName())) {
                 Logger.getGlobal().log(Level.INFO, "Adding excluded groups to existing ones");
                 String current = config.getValue();
+                // Should not add duplicate entry for NonDexIgnore
+                if (current.contains("edu.illinois.NonDexIgnore")) {
```

Comment:
I see that you remove `*Ignore` related code from some ITs, but you are adding a special case here. Why the need to remove from the `pom.xml` files in those ITs?